src/s3/explorer/s3FolderNode.ts

```diff
@@ -67,7 +66,7 @@ export class S3FolderNode extends AWSTreeNodeBase implements AWSResourceNode, Lo
         this.childLoader.clearChildren()
     }
 
-    private async loadPage(continuationToken: string | undefined): Promise<ChildNodePage> {
+    private async loadPage(continuationToken: string | undefined): Promise<ChildNodePage<S3FolderNode | S3FileNode>> {
```

Comment:
This return type _is_ needed (or it can be added to the `childLoader` declaration) because this class is self-referential; we cannot infer types in this case.